magefile.go

```diff
@@ -9,36 +9,23 @@ import (
 	"os"
 	"os/exec"
 	"path/filepath"
-
-	"github.com/magefile/mage/target"
 )
 
 var Default = Proto
 
 // regenerate protobuf
 func Proto() error {
-	updated, err := target.Path("proto/livekit_models.pb.go",
-		"livekit_internal.proto",
-		"livekit_models.proto",
-		"livekit_recording.proto",
-		"livekit_room.proto",
-		"livekit_rtc.proto",
-		"livekit_webhook.proto",
-		"livekit_analytics.proto",
-	)
-	if err != nil {
-		return err
+	twirpProtoFiles := []string{
+		"proto/livekit_recording.proto",
+		"proto/livekit_room.proto",
 	}
-	if !updated {
-		return nil
-	}
```

Comment:
I have removed the dirtyness check.  We do not practice it in other projects for either wire nor protobuf generation, and it feels dangerous.

I did add more explicit lists of source files.